convert2rhel/pkghandler.py

```diff
@@ -1011,14 +1020,16 @@ def get_total_packages_to_update():
     if it's Oracle/CentOS Linux 7 or 6, it will use `yum`, otherwise it should use `dnf`.
 
     :return: The packages that need to be updated.
-    :rtype: List[str]
+    :rtype: list[str]
     """
     packages = []
 
+    reposdir = get_eus_repos_available()
+
     if pkgmanager.TYPE == "yum":
         packages = _get_packages_to_update_yum()
     elif pkgmanager.TYPE == "dnf":
-        packages = _get_packages_to_update_dnf()
+        packages = _get_packages_to_update_dnf(reposdir=reposdir)
```

Comment:
```suggestion
        # We're using the reposdir with dnf only because we currently hardcode the repofiles for RHEL 8 derivatives only.
        packages = _get_packages_to_update_dnf(reposdir=reposdir)
```

Reply:
done.